lib/Service/ProviderService.php

```diff
@@ -79,6 +79,7 @@ class ProviderService
                 'id' => 'appid',
                 'secret' => 'secret',
             ],
+            'group_mapping' => 'groupMapping',
```

Comment:
I don't understand what `$configMapping` is about. We control the schema, can't we just force all the providers to use the same one?

Reply:
hybridauth is external lib

Reply:
It just works as it should. I think with the little Tutorial, everyone can set this up!